django/contrib/admin/templatetags/admin_list.py

```diff
@@ -122,7 +122,8 @@ def result_headers(cl):
                 # Not sortable
                 yield {
                     "text": text,
-                    "class_attrib": format_html(' class="column-{}"', field_name),
+                    "help_text": cl.model_admin.get_field_help_text(field_name),
+                    "class_attrib": format_html('class="column-{}"', field_name),
```

Comment:
Original code had space prepended, so I'd suggest to leave it and remove space in template `django/contrib/admin/templates/admin/change_list_results.html`